scripts/build.js

```diff
@@ -18,9 +19,18 @@ const FILES_TO_COPY = [
 
 shell.echo(`Start building...`);
 
-shell.mkdir(`${NPM_DIR}`);
+shell.mkdir(`-p`, `${NPM_DIR}`);
 shell.rm(`-Rf`, `${NPM_DIR}/*`);
```

Comment:
I know it shouldn't happen, but can we put a check here that NPM_DIR has some length?  I'm always nervous around a script that can run `rm -Rf /*`.